org.OpenT2T.Sample.SuperPopular.Lamp/Philips Hue Light Bulb/ts/thingTranslator.ts

```diff
@@ -0,0 +1,108 @@
+
+declare function require(module: string): any;
```

Comment:
Import doesn't work when there are no type definitions. I could have created (partial) type definitions for node-hue-api, but I wanted to demonstrate how that isn't necessary.